gramex/data.py

```diff
@@ -1498,14 +1499,178 @@ def _insert_mongodb(url, rows, meta=None, database=None, collection=None, **kwar
     return len(result.inserted_ids)
 
 
+def _get_influxdb_schema(client, bucket):
+    imports = 'import "influxdata/influxdb/schema"\n'
+    meas = client.query_api().query(
+        imports + f'schema.measurements(bucket: "{bucket}")'
+    )[0]
+    tags = client.query_api().query(imports + f'schema.tagKeys(bucket: "{bucket}")')[0]
+    tags = [r.get_value() for r in tags.records]
+    tags = [r for r in tags if not r.startswith("_")]
+    fields = client.query_api().query(
+        imports + f'schema.fieldKeys(bucket: "{bucket}")'
+    )[0]
+    return {
+        "_measurement": [r.get_value() for r in meas.records],
+        "_tags": tags,
+        "_fields": [r.get_value() for r in fields.records]
+    }
+
+
+_influxdb_op_map = {"<~": "<=", ">~": ">=", "!": "!=", "": "=="}
+
+
+def _influxdb_offset_limit(controls):
+    offset = controls.pop("_offset", ["-30d"])[0]
+    limit = controls.pop("_limit", [False])[0]
```

Comment:
Does this offset and limit default give us the LATEST 10,000 records? That's what we need.